### DISCUSSION TITLE
Harbfuzz/GTK 4 compilation with Devuan

### DISCUSSION THREAD

**Discussion by pincusp** [harfbuzz/harfbuzz#4418] [Q&A] [answered]
Hello,

I am trying to compile harbfuzz in order to obtain GTK 4. I tried with versions 8.2.11 and with 2.7.4 (as explained by BrandAccount on youtube), however my Libreoffice is stopped from starting with the following error message (also Thunderbird was hold up by a similar error, but I was able to make it start again):

![harbfuzz01](https://github.com/harfbuzz/harfbuzz/assets/145856637/fab3c3d3-aec3-4e2a-976a-4b5011939f13)

On an old manual for harbfuzz (harbfuzz.github.io/harbfuzz-hb-graphite2-html), I read that for graphite2 to work, harfbuzz must be compiled with the graphite2 engine enabled and that the default is not to enable it.

Does this have something to do with this error message?

How to compile harbfuzz with the graphite2 engine enabled?

Thanks.

**Comment by behdad** [comment, marked as answer]
> Does this have something to do with this error message?

Yes.

> How to compile harbfuzz with the graphite2 engine enabled?

If you are using the `meson` build system, reconfigure with `-Dgraphite2=enabled`.
